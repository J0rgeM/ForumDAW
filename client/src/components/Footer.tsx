import React from 'react';

function Footer() {
    return (
        <div className="footer text-center fixed-bottom text-white bg-dark">
            <div className="container-fluid">
                <div className="row">
                    <p className="text-center">Â© 2023 DAW</p>
                </div>
            </div>
        </div>
    );
}

export default Footer;
