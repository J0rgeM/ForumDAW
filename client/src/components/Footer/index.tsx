import React from 'react';

function Index() {
    return (
        <div className="footer text-center fixed-bottom text-white bg-dark">
            <div className="container-fluid">
                <div className="row">
                    <p className="text-center">Â© 2023 DAW</p>
                </div>
            </div>
        </div>
    );
}

export default Index;
