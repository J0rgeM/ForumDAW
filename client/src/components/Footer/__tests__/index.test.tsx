import {describe, expect, it} from '@jest/globals';
import {render} from "@testing-library/react";
import Footer from ".."
// describe serve para descrever o teste

describe('Footer', () => {
    it('match snapshot', () => {
        const { container } = render(<Footer/>) // container vai conter o snapshot
        expect(container).toMatchSnapshot()
    })
})

