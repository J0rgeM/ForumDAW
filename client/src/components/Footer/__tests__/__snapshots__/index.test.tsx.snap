// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer match snapshot 1`] = `
<div>
  <div
    class="footer text-center fixed-bottom text-white bg-dark"
  >
    <div
      class="container-fluid"
    >
      <div
        class="row"
      >
        <p
          class="text-center"
        >
          Â© 2023 DAW
        </p>
      </div>
    </div>
  </div>
</div>
`;
